tests/Fixer/FunctionNotation/NoUselessSprintfFixerTest.php

```diff
@@ -94,10 +94,16 @@ function sprintf($foo) {
                 echo sprint("%d", 1);
                 echo sprint("%d%d%d", 1, 2, 3);
                 echo sprint();
-                echo sprint[2]("foo");
```

Comment:
i extracted this one to https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5323 , hoping it will make CI green again

Reply:
i merged 5323, please, describe what is the remaining part of this PR and we can go further with it ;)